pkg/operation/seed/scheduler/gardener_kube_scheduler_bootstrap.go

```diff
@@ -0,0 +1,137 @@
+
+import (
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/features"
+	gardenletfeatures "github.com/gardener/gardener/pkg/gardenlet/features"
+	"github.com/gardener/gardener/pkg/operation/botanist/component"
+	"github.com/gardener/gardener/pkg/operation/seed/scheduler/configurator"
+	schedulerconfigv18 "github.com/gardener/gardener/pkg/operation/seed/scheduler/v18"
+	schedulerconfigv19 "github.com/gardener/gardener/pkg/operation/seed/scheduler/v19"
+	schedulerconfigv1alpha2 "k8s.io/kube-scheduler/v18/v1alpha2"
+	schedulerconfigv1beta1 "k8s.io/kube-scheduler/v19/v1beta1"
+
+	"github.com/Masterminds/semver"
+	"github.com/pkg/errors"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	"k8s.io/utils/pointer"
+)
+
+// Bootstrap is used to bootstrap gardener-kube-scheduler in Seed clusters.
+func Bootstrap(
+	c kubernetes.Interface,
+	image string,
+	seedVersion *semver.Version,
+) (component.DeployWaiter, error) {
+	var (
+		config configurator.Configurator
+		err    error
+	)
+	const namespace = "gardener-kube-scheduler"
+
+	if seedVersion == nil {
+		return nil, errors.New("seedVersion is required")
+	}
+
+	switch seedVersion.Minor() {
+	case int64(18):
+		config, err = schedulerconfigv18.NewConfigurator(namespace, &schedulerconfigv1alpha2.KubeSchedulerConfiguration{
+			Profiles: []schedulerconfigv1alpha2.KubeSchedulerProfile{{
+				SchedulerName: pointer.StringPtr(GardenerShootControlPlaneSchedulerName),
+				Plugins: &schedulerconfigv1alpha2.Plugins{
+					Score: &schedulerconfigv1alpha2.PluginSet{
+						Disabled: []schedulerconfigv1alpha2.Plugin{
+							{Name: "NodeResourcesLeastAllocated"},
+							{Name: "NodeResourcesBalancedAllocation"},
+						},
+						Enabled: []schedulerconfigv1alpha2.Plugin{
+							{Name: "NodeResourcesMostAllocated"},
+						},
+					},
+				},
+			}},
+		})
+	case int64(19):
+		config, err = schedulerconfigv19.NewConfigurator(namespace, &schedulerconfigv1beta1.KubeSchedulerConfiguration{
+			Profiles: []schedulerconfigv1beta1.KubeSchedulerProfile{{
+				SchedulerName: pointer.StringPtr(GardenerShootControlPlaneSchedulerName),
+				Plugins: &schedulerconfigv1beta1.Plugins{
+					Score: &schedulerconfigv1beta1.PluginSet{
+						Disabled: []schedulerconfigv1beta1.Plugin{
+							{Name: "NodeResourcesLeastAllocated"},
+							{Name: "NodeResourcesBalancedAllocation"},
+						},
+						Enabled: []schedulerconfigv1beta1.Plugin{
+							{Name: "NodeResourcesMostAllocated"},
+						},
+					},
+				},
+			}},
+		})
+	default:
+		config = configurator.NoOp()
+	}
+
+	if err != nil {
+		return nil, err
+	}
+
+	scheduler, err := New(
+		c.DirectClient(),
+		namespace,
+		image,
+		seedVersion,
+		config,
+		&admissionregistrationv1beta1.WebhookClientConfig{
+			Service: &admissionregistrationv1beta1.ServiceReference{
+				Name:      "gardener-seed-admission-controller",
```

Comment:
Export https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/seedsystemcomponents/seedadmission/seedadmission.go#L48 and use it here